com.unity.ml-agents/Runtime/SideChannels/TrainingAnalyticsSideChannel.cs

```diff
@@ -1,3 +1,8 @@
+#if MLA_UNITY_ANALYTICS_MODULE || !UNITY_2019_4_OR_NEWER
+#define MLA_UNITY_ANALYTICS_MODULE_ENABLED
+#endif
+
+#if MLA_UNITY_ANALYTICS_MODULE_ENABLED
```

Comment:
can remove this

Reply:
This needs to be undone - we should always create the sidechannel to listen on, even if we don't use it.